src/debugger/adapter/DebugAdapter.js

```diff
@@ -114,10 +114,9 @@ class PrepackDebugSession extends DebugSession {
     let adapterChannel = new AdapterChannel(inFilePath, outFilePath);
     this._adapterChannel = adapterChannel;
     this._registerMessageCallbacks();
-    let separatedSourceFiles = args.sourceFile.trim().split(/\s+/g); // Split on all whitespace
     let launchArgs: PrepackLaunchArguments = {
       kind: "launch",
-      sourceFiles: separatedSourceFiles,
+      sourceFiles: args.sourceFiles,
```

Comment:
Do you want to inline all fields of `args` into `launchArgs`? If so, these three lines can be simplified using spread operation: `...args`